teaching-platform/exceptions.py

```diff
@@ -0,0 +1,56 @@
+import logging
+
+
```

Comment:
I suggest adding hierarchy to your exceptions. The number of exceptions will grow during development process but, for example - if you add parent exception class `UserPolicyNotMet(Exception)` (or something similar) and change your existing exceptions:
```python
class PasswordToWeakError(UserPolicyNotMet):

class PasswordToWeakError(UserPolicyNotMet):

```
Then you will be able to handle all set of similar exceptions with single
```python
except PolicyNotMetError
```

When new constraint will be added to user account - there is a good chance that keeping this hierarchy will result it less code changes overall.

Reply:
Done. It's for sure going to help when there's a lot more exceptions.